pwiz_tools/BiblioSpec/src/BuildParser.cpp

```diff
@@ -23,12 +23,13 @@
 #include "pwiz/utility/misc/Filesystem.hpp"
 #include "BuildParser.h"
 #include "SpecData.h"
+#include "SslReader.h"
 
 namespace BiblioSpec {
 
 BuildParser::BuildParser(BlibBuilder& maker,
-                         const char* filename,
-                         const ProgressIndicator* parentProgress_)
+                             const char* filename,
+                             const ProgressIndicator* parentProgress_)

```

Comment:
Spurious whitespace change?

Reply:
Yeah, that's evil, thanks for pointing it out